src/main/java/model/Thing.java

```diff
@@ -1,7 +1,16 @@
 package model;
 
 public class Thing {
-    String name;
+    private Long id;
+    private String name;
```

Comment:
What is `thing`? @mich160 is it correctly to give a names which are not related with object?

Reply:
Agree, the name could be better. This seems to generic, what about `contribution`?